### ISSUE TITLE
Error during ./INSTALL.sh

### ISSUE THREAD

**Issue by jialuli-luka** [google/clif#37]
Hi, I meet this error while installing the cliff:
CMake Error at clif/cmake/modules/CLIFUtils.cmake:35 (find_package):
  By not providing "Findabsl.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "absl", but
  CMake did not find one.

  Could not find a package configuration file provided by "absl" with any of
  the following names:

    abslConfig.cmake
    absl-config.cmake

Any help would be appreciated!

**Comment by jialuli-luka** [comment]
I solve this problem after using old commit f151944.

However, another problem occurred. 

Traceback (most recent call last):
  File "/playpen3/home/jialuli/touchdown/clif_backend/llvm/tools/clif/pyclif.py", line 37, in <module>
    from clif.protos import ast_pb2
  File "/playpen3/home/jialuli/touchdown/clif_backend/build_matcher/tools/clif/protos/ast_pb2.py", line 5, in <module>
    from google.protobuf import descriptor as _descriptor
ImportError: No module named google.protobuf

I check with pip show protobuf, it gives:
Name: protobuf
Version: 3.13.0
Summary: Protocol Buffers
Home-page: https://developers.google.com/protocol-buffers/
Author: None
Author-email: None
License: 3-Clause BSD License
Location: /playpen3/home/jialuli/python3/lib/python3.6/site-packages
Requires: six, setuptools
Required-by: tensorboardX, tensorboard

Any help would be appreciated!

**Comment by akekawit** [comment]
It looks like our documentation did not mention that you now need LLVM/Clang 11, absl-cpp and googletest. I have just updated the README in f0fd895. Could you try again with the latest commit on main with these prerequisites installed?

**Comment by jialuli-luka** [comment]
Thanks for the reply! I installed abseil in the subdirectory of clif. Can I know how to link abseil with CMakefiles in clif? I tried add_subdirectory(abseil-cpp) but it doesn't work.

**Comment by akekawit** [comment]
If you have abseil installed in a non-traditional path, you might have to play around with CMake to make sure that it's searching that directory. There are couple of ways to do this based on CMake's search procedures: https://cmake.org/cmake/help/latest/command/find_package.html#search-procedure.

If you need to incorporate abseil into the CMake project for some reasons (with add_subdirectory, etc.), there might be additional steps that you need to do. Check out https://github.com/google/googletest/blob/master/googletest/README.md#incorporating-into-an-existing-cmake-project for googletest. It should be very similar for abseil.

The simplest solution I think is to grab one of the [abseil releases](https://github.com/abseil/abseil-cpp/releases/) and just run make install. See [Dockerfile](https://github.com/google/clif/blob/main/Dockerfile) for an example.

**Comment by rwgk** [comment]
Closing. Please open a new issue as needed.
